src/main/java/frc/robot/subsystems/elevator/ElevatorInputs.java

```diff
@@ -17,4 +14,6 @@ public class ElevatorInputs {
     public ElevatorIO.ControlMode controlMode = ElevatorIO.ControlMode.POSITION;
     public boolean isBottom = true;
     public boolean isTop = false;
+    public MutableMeasure<Angle> servoAngle = Units.Degrees.of(0).mutableCopy();
+    public MutableMeasure<Angle> servoSetpoint = Units.Degrees.of(0).mutableCopy();
```

Comment:
The names should be a bit closer to the "problem space", so instead of servo, use the name of the mechanism using this servo.